projects/dsp-ui/src/lib/search/advanced-search/advanced-search.component.ts

```diff
@@ -268,10 +273,13 @@ export class AdvancedSearchComponent implements OnInit, OnDestroy {
             }
         );
 
-        const gravsearchQuery = this._gravsearchGenerationService.createGravsearchQuery(properties, resClass);
+        const gravsearch: SearchParams = {
+            query: this._gravsearchGenerationService.createGravsearchQuery(properties, resClass),
```

Comment:
it could be declare in a variable instead of here directly - e.g. expert-search (to make both component consistent)

Reply:
Done in f0268ac